controllers/page.js

```diff
@@ -60,20 +68,22 @@ exports.deletePage = (req, res, next) => {
  * Update page
  */
 exports.putPage = (req, res, next) => {
-  const siteId = req.params.id;
+  const site = req.params.id;
   const pageId = req.params.pageId;
 
-  Page.findOne({ _id: pageId, siteId }).then((page) => {
+  Page.findOne({ _id: pageId, site }).then((page) => {
     if (!page) {
       return res.json({ message: 'Page not found' });
     }
 
-    page.siteId = req.body.siteId || page.siteId;
+    page.site = req.body.site || page.site;
     page.authorId = req.user._id || page.authorId;
     page.title = req.body.title || page.title;
     page.content = req.body.content || page.content;
-    page.isPublished = req.body.isPublished || page.isPublished;
+    page.isPublished = req.body.is_published || page.isPublished;
     page.slug = slug(req.body.slug, { lower: true, charmap: '' }) || page.slug;
```

Comment:
`slug(req.body.slug, { lower: true, charmap: '' })` should only be executed when `req.body.slug` is exist. Otherwise it will throw error.